maintner/github.go

```diff
@@ -143,6 +143,31 @@ func (gr *GitHubRepo) ForeachMilestone(fn func(*GitHubMilestone) error) error {
 	return nil
 }
 
+// ErrIssueNotFound represents the case where an issue is not found.
+var ErrIssueNotFound = fmt.Errorf("issue not found.")
+
+// GetIssue returns a GitHubIssue with the given ID.
+//
+// It returns nil if no issue with the given ID exists
```

Comment:
What if the issue is in the map but has been tombstoned (marked as not_exists)?

Reply:
Our downstream code handles that by also checking the .NotExist field. Could be added here as well

Reply:
Okay, if `Foreachissue` returns NotExist issues then this should too.